doc/schema_template.md

```diff
@@ -10,11 +10,9 @@ If you find a bug please report it [here](https://github.com/galaxyproject/galax
 
 This document serves as reference documentation. If you would like to learn
 how to build tools for Galaxy,
-[Planemo](http://planemo.readthedocs.io/en/latest/writing.html) features a
+[Planemo](https://planemo.readthedocs.io/en/latest/writing.html) features a
 number of tutorials on building Galaxy tools that would better serve that purpose.
 
-$toc
```

Comment:
No, this doesn't work now that we go through rst. As mentioned above, the rst created by pandoc contains the TOC command at the top:
```
.. contents::
  :depth: 2.0
..
```
but Sphinx for unknown reasons (old version maybe) ignores it on Jenkins while it works locally for both @jmchilton and me.

Please note that the commands above are general rst, while Sphix has also its own syntax: http://www.sphinx-doc.org/en/1.4.8/markup/toctree.html